README.md

```diff
@@ -43,7 +43,7 @@ If some changes are not visible (after saving a page while editing), please rest
 ## Important points to be noted
 
 1. Whenever the code is changed, please ensure that the relevant code changes are reflected in **main.js** file (For example, if a slideshow is disabled, make sure the id of the slideshow element/section is also disabled in the **main.js** file if it exists).
-2. Whenever a new commit is made and the changes are made to any one of airspace.css, main.js, or plugins.js files, try to use a minification service and add the minified version of new code in respective **minified** folder. (css and js folders have a **minified** folder)
+2. HTML files read from minified CSS and JS files. While making changes to unminified files, you should use `grunt` to watch changes to the unminified files, and grunt will automatically minify them as needed. Remember to `npm install` before install necessary dependencies before running `grunt`. 
```

Comment:
Imperative phrasing needed?  After making changes to some files, use grunt to update the minified file.